docs/0.2-DRAFT/index.md

```diff
@@ -171,21 +171,20 @@ RO-Crate relies heavily on [schema.org](http://schema.org) using a constrained s
 
 Generally, the standard keys for [schema.org](http://schema.org) should be used. However, RO-Crate uses variant names for some elements, specifically:
 
-
 *   `File` is mapped to <http://schema.org/MediaObject> which was chosen as a compromise as it has many of the properties that are needed to describe a generic file. Future versions of schema.org or a research data extension may re-define `File`.
 *   `Journal` is mapped to <http://schema.org/Periodical>.
 *   `path` is mapped to <http://schema.org/contentUrl>. This property is used on some classes which do not strictly (yet) allow it in the official Schema.org.
 
 
-## Additional metadata standards
+### Additional metadata standards
 
 The following terms from the from the ro-terms ontology <http://purl.org/ro/wf4ever#> are used:
 
-- `WorkflowSketch` for an image depicting a workflow is , see the [documentation](http://wf4ever.github.io/ro/2016-01-28/roterms/#Sketch.)
+- `WorkflowSketch` for an image depicting a workflow, see the [documentation](http://wf4ever.github.io/ro/2016-01-28/roterms/#Sketch.)
 
-- `WorkfloScript` for a script in a  [documentation](http://wf4ever.github.io/ro/2016-01-28/roterms/#Script)
+- `WorkfloScript` for a script in a [documentation](http://wf4ever.github.io/ro/2016-01-28/roterms/#Script)
```

Comment:
Typo in `WorkflowScript`. I would try to avoid multiple links called [documentation](#) (not skimmable) and rather say something like 

_`WorkflowSketch` for an image depicting a workflow, mapped to [roterms:Sketch](http://wf4ever.github.io/ro/2016-01-28/roterms/#Sketch)._